project_change_state/wizard/project_change_state.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openerp import api, models
+
+
+class ProjectChangeState(models.TransientModel):
+    _name = "project.change.state"
+    _description = "Change the status of the selected projects"
+
+    @api.multi
+    def set_pending(self):
+        project_ids = self.env.context.get('active_ids')
+        for project in self.env['project.project'].browse(project_ids):
+            project.set_pending()
```

Comment:
No need for a for-loop. Just use: 

    self.env['project.project'].browse(project_ids).set_pending()

Or:

    projects = self.env['project.project'].browse(project_ids)
    projects.set_pending()